CAIPs/caip-261.md

```diff
@@ -0,0 +1,466 @@
+  - **did:key:** This method allows for the generation of DIDs directly from cryptographic public keys, supporting various key types such as ECDSA, EdDSA, Schnorr Signatures, and RSA. It is straightforward and self-contained, with no need for interaction with blockchain networks. This makes it fast and cost-effective for scenarios that do not require decentralized verification.
+- **On-chain Public Accounts** using the chain-specific address type for smart contract-based assertions (e.g., `address` type in EVM-based chains) 
+
+##### Assertions Identification
+
+Assertions themselves need to be identifiable for referencing or updating purposes. 
+They SHOULD be identified by a immutable identifier.
+- **[Content Identifiers (CID)][CID]**, which is self-describing and facilitates their unique identification and retrieval. If a CID is used, the CID SHOULD be envelopped as a Uniform Resource Identifier (URI) to offer a contextual identifier, such as IPFS (ipfs://).
+  - This is not necessarily embedded as a property within the document itself; it derives from the content of the document and can be calculated (or recalculated, as a checksum of its contents) by anyone.
+  - This is generated by hashing the content with a cryptographic hash function; This hash, along with metadata about the hash function, is then encoded in the [multihash][] format.
+- **On-chain custom identifiers** generated and assigned by a smart contract based assertion storage, i.e. on-chain attestations identifiers (e.g., Verax, EAS). 
+
+#### Data Model
+
+Subsequent data structures are represented in accordance with the [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/) and, in some cases, in raw format relevant for on-chain assertions. 
+
+Verifiable credentials can be used both off-chain and on-chain for better portability, while raw formats can be utilized in decentralized data storages (e.g., Ceramic Network) or on-chain assertion storages (e.g., Verax, EAS) as a new schema to issue assertions without the need for additional layers of verifiability and interoperability.
+
+However, this standard does not prescribe any specific document type, though it recommends using internationally recognized standards or trusted attestation services."
+
+![diagram2](https://github.com/dayksx/CAIPs/assets/77788154/479df06f-3dd5-4c88-b396-4cfae01ff8d6)
+
+**Peer Trust Assertions:**
+
+Modeling trust and distrust towards an individual or entity can be a complex task due to the subjective and multifaceted nature of trust.
+This standard proposes the following conceptualization for the trust concept:
+
+- `scope`: This defines the applicable trust perimeter. It SHOULD be a noun. Any implementation or profile of this specification COULD enumerate or provide reference lists of valid "scope" values to facilitate interoperability across different systems.

```

Comment:
```suggestion
- `scope`: This defines the applicable trust perimeter. It SHOULD be a noun. Any implementation or profile of this specification SHOULD enumerate or provide reference lists of valid "scope" values to facilitate interoperability across different systems.
```
The IETF options are only SHOULD and MAY, not COULD; also, I feel strongly that it's a SHOULD here, not a MAY.

https://www.rfc-editor.org/rfc/rfc2119